dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubHelper.java

```diff
@@ -32,7 +58,8 @@
 
 public final class GitHubHelper {
 
-    private static final Logger LOG = LoggerFactory.getLogger(GoogleHelper.class);
+    private static final Logger LOG = LoggerFactory.getLogger(GitHubHelper.class);
+    private static final OkHttpClient OK_HTTP_CLIENT = new OkHttpClient(); // OkHttpClient JavaDoc recommends using one instance
```

Comment:
Not sure what the Javadoc reasoning is, but it is also true for us. 
The client is created once with a cache in https://github.com/dockstore/dockstore/blob/develop/dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceApplication.java#L194 
This is creating a new one without a cache. This should re-use that to share a cache and keep rate limit low.

Reply:
The comment was because the code was previously creating a new instance in every method invocation, and I wanted to explain why I had moved it out of the method.

But, even better to use the existing instance and cache from DockstoreWebServiceApplication -- fixed in next push.